packages/libs/components/src/plots/BipartiteNetwork.tsx

```diff
@@ -107,18 +109,14 @@ function BipartiteNetwork(
   // nodes based on their column, then will use their order in the column
```

Comment:
mentions of "column" should be updated with "partition", or at least we should explain somewhere how they relate